frontend/src/components/App/AppSidebar/RerollButton.jsx

```diff
@@ -33,27 +33,27 @@ class RerollButton extends PureComponent {
 
     render() {
         let users,
-            skips;
+            rerolls;
 
-        if (this.props.connectedUser !== null && this.props.connectedUser !== undefined) {
+        if (this.props.connectedUser) {
             users = this.props.connectedUser.length + 1;
-            skips = 0;
+            rerolls = 0;
             for (let j = 0; j < this.props.connectedUser.length; j++) {
-                if (this.props.connectedUser[j].voted === 'skip') {
-                    skips++;
+                if (this.props.connectedUser[j].voted === 'reroll') {
+                    rerolls++;
                 }
             }
         }
 
-        if (this.props.host !== null && this.props.host !== undefined) {
-            if (this.props.host.voted === 'skip') {
-                skips++;
+        if (this.props.host) {
+            if (this.props.host.voted === 'reroll') {
```

Comment:
```suggestion
        if (this.props.host && this.props.host.voted === 'reroll') {
```

Reply:
Sry before i understood the difference between `&` and `&&`.